### PR TITLE
update wg governance

### PR DESCRIPTION
**Pull request by gingerkidney**
Signed-off-by: notginger <zhangyanqing@pingcap.com>

### CODE REVIEW

**Comment by gingerkidney** [comment]
@dcalvin PTAL

File: committee/wg-governance/wg-governance.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+The SIGs related to this Working Group should be listed on the Working Group's page in the `Stakeholder SIGs` section. 
+
+## WG Lifecycle 
+
+### Create
+
+* Determine WG organizers and members to ensure that members are TiKV Contributor at least.
+* Create a `xyz-wg` directory in [wg directory](/wg) (modify xyz to your WG name) according to [Working Group template](working-group-template.md), and modify the Working Group template as `README.md` page
+* Submit a Pull Request on [tikv/community](https://github.com/tikv/community) , which is determined by voting by the Maintainer/PMC. The Working Group is announced after the Pull Request is merged.
+* Create a channel corresponding to the Working Group on [Slack: tikv-wg.slack.com](tikv-wg.slack.com) .
```
Comment (dcalvin) [inline committee/wg-governance/wg-governance.md]: Suggest re-write this section as below:

* Before creating a working group, make sure you get the following ready:
    * Ensure all WG Organizers, and other leadership roles are community members
    * Have the answers to following questions:
        * What is the exact problem this group is trying to solve?
        * What is the artifact that this group will deliver, and to whom?
        * How does the group know when the problem solving process is completed, and it is time for the Working Group to dissolve?
        * Who are all of the stakeholder SIGs involved in this problem this group is trying to solve? Have you 
        * What are the meeting mechanics (frequency, duration, roles)?
* Once you have met the above criteria, you can submit a Pull Request titled “"WG-Creation-Request: XXX WG”, with all the items  and related documents (RFCs, PRs) attached. The PR will be reviewed by the PMC. A supermajority vote is required for the PR to pass.
* Create a `xyz-wg` directory in [wg directory](/wg) (modify xyz to your WG name) according to the requirements defined in [Working Group template](working-group-template.md), and modify the Working Group template accordingly.
*  Once the PR is approved, this WG is properly charted. You can create a channel corresponding to the Working Group on [Slack: tikv-wg.slack.com](tikv-wg.slack.com)

File: committee/wg-governance/wg-governance.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+* Determine WG organizers and members to ensure that members are TiKV Contributor at least.
+* Create a `xyz-wg` directory in [wg directory](/wg) (modify xyz to your WG name) according to [Working Group template](working-group-template.md), and modify the Working Group template as `README.md` page
+* Submit a Pull Request on [tikv/community](https://github.com/tikv/community) , which is determined by voting by the Maintainer/PMC. The Working Group is announced after the Pull Request is merged.
+* Create a channel corresponding to the Working Group on [Slack: tikv-wg.slack.com](tikv-wg.slack.com) . 
+
+### Dismiss
+
+* The Working Group can be dismissed by the Tech Lead after mission completed.
+* If the working group 's work has stalled, the PMC can also vote to dismiss the working group .
```
Comment (dcalvin) [inline committee/wg-governance/wg-governance.md]: "has stalled" is not a solid and specific reason for dismissing the group.

Comment (dcalvin) [reply committee/wg-governance/wg-governance.md]: A WG can also dissolve voluntarily

File: committee/wg-governance/wg-governance.md [hunk 3]
```diff
@@ -1 +1,42 @@
-TBD: will be finished after [wg-governance-zh_CN.md] is merged.
\ No newline at end of file
+# TiKV Working Group Governance
+
+## What is Working Group?
+
+The Working Group provides a group to allow community contributors to collaborate for a period of time on an issue. Due to the Working Group usually solves a problem that spans the scope of responsibility of multiple Special Interest Groups (SIGs), so it usually consists of members from different SIGs or some other community contributors. The Working Group usually dismisses after the problem is solved. And the results or members in the Working Group can be accepted by the SIG after. For more detailed procedures of creating or dissmiss, see [WG Lifecycle](#wg-lifecycle).
```
[the remark this answers is not in the document]
Comment (Hoverbear) [reply committee/wg-governance/wg-governance.md]: Perhaps the first sentence could be:

> A working Group (WG) is a formal recognition of a temporary cooperation of a group of contributors working towards a shared specific goal.

File: committee/wg-governance/wg-governance.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+
+The Working Group has the following characteristics: 
+
+* Does not own the code
+* Have clear measurable goals and results that can be delivered
+* Is a short-term organization and will be disbanded after the goal or mission is completed
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]: Suggest adding "Typically addresses a concern affecting multiple SIGs or subprojects."

File: committee/wg-governance/wg-governance.md [hunk 5]
```diff
@@ -1 +1,42 @@
...
+### Create
+
+* Determine WG organizers and members to ensure that members are TiKV Contributor at least.
+* Create a `xyz-wg` directory in [wg directory](/wg) (modify xyz to your WG name) according to [Working Group template](working-group-template.md), and modify the Working Group template as `README.md` page
+* Submit a Pull Request on [tikv/community](https://github.com/tikv/community) , which is determined by voting by the Maintainer/PMC. The Working Group is announced after the Pull Request is merged.
+* Create a channel corresponding to the Working Group on [Slack: tikv-wg.slack.com](tikv-wg.slack.com) . 
+
+### Dismiss
+
+* The Working Group can be dismissed by the Tech Lead after mission completed.
```
[the remark this answers is not in the document]
Comment (gingerkidney) [reply committee/wg-governance/wg-governance.md]: Not SIG, organizer should be more accurate

**Comment by gingerkidney** [comment]
@Hoverbear PTAL

File: committee/wg-governance/wg-governance.md [hunk 6]
```diff
@@ -1 +1,57 @@
-TBD: will be finished after [wg-governance-zh_CN.md] is merged.
\ No newline at end of file
+# TiKV Working Group Governance
+
+## What is Working Group?
+
+Contributors are totally able to just work on goals together without it being a WG or formal. We're not 'allowing' it, this is just a formal recognition that this will happen and that WGs can empower and enhance these cooperations.
+
+Due to the Working Group usually solves a problem that spans the scope of responsibility of multiple Special Interest Groups (SIGs), so it usually consists of members from different SIGs or some other community contributors. The Working Group usually dismisses after the problem is solved. And the results or members in the Working Group can be accepted by the SIG after. For more detailed procedures of creating or dismiss, see [WG Lifecycle](#wg-lifecycle).
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]:
```suggestion
A Working Group usually solves a problem that spans the scope of responsibility of multiple Special Interest Groups (SIGs), so it usually consists of members from different SIGs or some other community contributors. The Working Group usually dissolves after the problem is solved. For more detailed procedures of creating or dismiss, see [WG Lifecycle](#wg-lifecycle).
```

Removed a sentence that seems to imply contributors cannot be in both a SIG and a WG

File: committee/wg-governance/wg-governance.md [hunk 7]
```diff
@@ -1 +1,57 @@
...
+* Does not own the code
+* Have clear measurable goals and results that can be delivered
+* Is a short-term organization and will be disbanded after the goal or mission is completed
+* Typically addresses a concern affecting multiple SIGs or subprojects
+
+## Relationship between Working Group and Special Interest Group
+
+![wg_sig_relationship](/media/governance/wg_sig_relationship.png)
+
+The assets owned by the TiKV Project (eg: code, documentation, blogs, regulations, etc.) are owned and managed by SIGs. However, there are assets that are specially owned and managed by Working Groups, such as:
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]:
```suggestion
The assets owned by the TiKV Project (eg: code, documentation, blogs, regulations, etc.) are owned and managed by SIGs, maintainers, and ultimately our PMC. However, there are assets that are specially owned and managed by Working Groups, such as:
```

File: committee/wg-governance/wg-governance.md [hunk 8]
```diff
@@ -1 +1,57 @@
...
+## What is Working Group?
+
+Contributors are totally able to just work on goals together without it being a WG or formal. We're not 'allowing' it, this is just a formal recognition that this will happen and that WGs can empower and enhance these cooperations.
+
+Due to the Working Group usually solves a problem that spans the scope of responsibility of multiple Special Interest Groups (SIGs), so it usually consists of members from different SIGs or some other community contributors. The Working Group usually dismisses after the problem is solved. And the results or members in the Working Group can be accepted by the SIG after. For more detailed procedures of creating or dismiss, see [WG Lifecycle](#wg-lifecycle).
+
+The Working Group has the following characteristics: 
+
+* Does not own the code
+* Have clear measurable goals and results that can be delivered
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]:
```suggestion
* Have clear measurable goals and results that can be delivered
```

We might want to rephrase this line

File: committee/wg-governance/wg-governance.md [hunk 9]
```diff
@@ -1 +1,57 @@
...
+    * How does the group know when the problem-solving process is completed, and it is time for the Working Group to dissolve?
+    * Who are all of the stakeholder SIGs involved in this problem this group is trying to solve? Have you
+    * What are the meeting mechanics (frequency, duration, roles)?
+* Once you have met the above criteria, you can submit a Pull Request titled "WG-Creation-Request: XXX-WG", with all the items and related documents (RFCs, PRs) attached. The PR will be reviewed by the PMC. 2 approvals are required for the PR to pass.
+
+* Create an xyz-wg directory in [wg directory](/wg) (modify xyz to your WG name) according to the requirements defined in [Working Group template](working-group-template.md), and modify the Working Group template accordingly.
+
+* Once the PR is approved, this WG is properly charted. You can create a channel corresponding to the Working Group on Slack: tikv-wg.slack.com
+
+### Dismiss
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]:
```suggestion
### Dissolution
```

File: committee/wg-governance/wg-governance.md [hunk 10]
```diff
@@ -1 +1,57 @@
...
+
+* The Working Group can be dismissed by the organizer after the mission completed.
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]:
```suggestion
* The Working Group can dissolve itself after it or our maintainers feel it's mission is complete.
```

File: committee/wg-governance/wg-governance.md [hunk 11]
```diff
@@ -1 +1,57 @@
...
+* If the working group has met one of the following requirements, the working group can be dismissed voluntarily:
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]:
```suggestion
* If the working group has met one of the following requirements, the working group dissolved by the PMC:
```

File: committee/wg-governance/wg-governance.md [hunk 12]
```diff
@@ -1 +1,57 @@
-TBD: will be finished after [wg-governance-zh_CN.md] is merged.
\ No newline at end of file
+# TiKV Working Group Governance
+
+## What is Working Group?
+
+Contributors are totally able to just work on goals together without it being a WG or formal. We're not 'allowing' it, this is just a formal recognition that this will happen and that WGs can empower and enhance these cooperations.
```
Comment (Hoverbear) [inline committee/wg-governance/wg-governance.md]:
```suggestion
Contributors are totally able to just work on goals together without it being a Working Group (WG) or formal team. A WG is a formal recognition that these kinds of groups will form, and that WGs can empower and enhance these cooperations.
```

**Comment by gingerkidney** [comment]
@Hoverbear PTAL again

File: committee/wg-governance/wg-governance.md [hunk 13]
```diff
@@ -1 +1,57 @@
...
+Working Groups have different goals with those of SIGs. A WG aims to:
+
+* Promote collaboration between different SIGs
+* Resolve some specific problems with a minimal organization
+
+The SIGs related to this Working Group should be listed on the Working Group's page in the `Stakeholder SIGs` section. 
+
+## WG Lifecycle 
+
+### Create
```
Comment (dcalvin) [inline committee/wg-governance/wg-governance.md]:
```suggestion
### Creation
```
To be consistent with Dissolution

**Pull request merged**